lucida/djinntonic/dig/Makefile

```diff
@@ -23,7 +23,7 @@ $(TARGET): $(OBJECTS)
 	$(CXX) -Wall $(CXXFLAGS) -c $< -o $@
 
 start_server: 
-	gnome-terminal -x bash -c "./DIGServer $(filter-out $@,$(MAKECMDGOALS)); read -n1"
+	./DIGServer
 
 start_test: 
 	gnome-terminal -x bash -c "./run-test.sh $(filter-out $@,$(MAKECMDGOALS)); read -n1"
```

Comment:
Okay, I created an issue. In the meantime let me look into our internal infrastructure.